stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/internal/MessageListScrollHelper.kt

```diff
@@ -76,6 +91,20 @@ internal class MessageListScrollHelper(
         )
     }
 
+    /**
+     * Checks if we have any popups shown over the list and stops scrolling in case we do.
+     *
+     * @param recyclerView The list that's being observed for long taps and scrolling.
+     */
+    private fun stopScrollIfPopupShown(recyclerView: RecyclerView) {
+        val fragmentManager = recyclerView.context.getFragmentManager() ?: return
+        val hasDialogsShown = fragmentManager.fragments.any { it is DialogFragment }
+
+        if (hasDialogsShown) {
+            recyclerView.stopScroll()
```

Comment:
This works pretty well. An alternative solution would be to use `LayoutManager.canScrollVertically`, but it is more verbose and requires introducing state. So, I think your solution is pretty neat.